openpype/pipeline/colorspace.py

```diff
@@ -601,3 +601,63 @@ def _get_imageio_settings(project_settings, host_name):
     imageio_host = project_settings.get(host_name, {}).get("imageio", {})
 
     return imageio_global, imageio_host
+
+
+def get_display_view_colorspace_name(config_path, display, view):
+    """Return colorspace name for the given display and view.
+
+    Args:
+        config_path (str): path string leading to config.ocio
+        display (str): display name e.g. "ACES"
+        view (str): view name e.g. "sRGB"
+
+    Returns:
+        view color space name (str) e.g. "Output - sRGB"
+    """
+
+    if not compatibility_check():
+        # python environment is not compatible with PyOpenColorIO
+        # needs to be run in subprocess
+        return get_display_view_colorspace_subprocess(config_path,
+                                                      display, view)
```

Comment:
is this the standard style indentation that you guys prefer at ynput?

Reply:
Seeing this from the sidelines I feel like the answer is "not per se". I'm seeing a lot of different contributions with a slight different style and usage of the indentation. I think at this point it's more like: If the hound doesn't complain it's fine.

```python
# A
        return get_display_view_colorspace_subprocess(config_path,
                                                      display, view)
# B
        return get_display_view_colorspace_subprocess(config_path,
                                                      display, 
                                                      view)
# C
        return get_display_view_colorspace_subprocess(
            config_path,
            display, 
            view
        )
```
Not sure if there's any preference on this really 'team-wide'.

Reply:
I like `B`! 

> If the hound doesn't complain it's fine.

Yeah, That's me :")

Reply:
the alignment on `A` and `B` only work if everyone uses the same indentation configuration (spaces/tabs and same size of indent) in their IDEs, that's why `C` is generally the preferred method